From 0c8e4bb5ef3b0df41bcfafb1cb16168023ee133e Mon Sep 17 00:00:00 2001
From: Hugo Osvaldo Barrera <hugo@whynothugo.nl>
Date: Mon, 19 Oct 2026 22:39:07 +0200
Subject: [PATCH] Fix test failing with urwid >= 4.2.5

Previously, a leading zero was cropped for focused field, and tests even
asserted this. Check the rendered value for a non-focused field, which does not
diverge across versions
---
 tests/ui/test_editor.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/ui/test_editor.py b/tests/ui/test_editor.py
index 13b6c2f..a4a599f 100644
--- a/tests/ui/test_editor.py
+++ b/tests/ui/test_editor.py
@@ -103,9 +103,9 @@ def test_convert_to_date():
         BERLIN.localize(dt.datetime(2017, 10, 4, 18)),
         conf=CONF,
     )
-    canvas = editor.render((50,), True)
+    canvas = editor.render((50,), False)
     assert CanvasTranslator(canvas, palette).transform() == (
-        "[ ] Allday\nFrom: \x1b[31m2.10.2017 \x1b[0m \x1b[34m13:00 \x1b[0m\n"
+        "[ ] Allday\nFrom: \x1b[34m02.10.2017\x1b[0m \x1b[34m13:00 \x1b[0m\n"
         "To:   \x1b[34m04.10.2017\x1b[0m \x1b[34m18:00 \x1b[0m\n"
     )
 
